scripts/circleci/update-snapshot-and-docs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+ARG_DEFS=(
+  "--sha=(.*)"
+)
+
+function init {
+  # If --git-push-dryrun or --verbose are set, be sure to export them
+  # so they are set in all the other scripts too
+  export GIT_PUSH_DRYRUN=$GIT_PUSH_DRYRUN
+  export VERBOSE=$VERBOSE
+}
+
+function run {
+  if [[ "$CIRCLE_PULL_REQUEST" != "false" ]]; then
```

Comment:
Running this file isn't enabled yet. I want to do some more testing and disable this in Travis before enabling it. I am not sure that this `CIRCLE_PULL_REQUEST` env var will work similar to `TRAVIS_PULL_REQUEST`.

Reply:
It should work similar to that. Though you need to either filter out branches or have a branch check here, because otherwise commits to upstream branches would cause this assertion to be true.